element_interface/utils.py

```diff
@@ -125,6 +127,13 @@ def ingest_csv_to_table(
         allow_direct_insert (bool): permit insertion into Imported and Computed tables
             See DataJoint's `insert`.
     """
+    if not isinstance(csvs, list):  # Ensure each input is a list
```

Comment:
It may not be a list. Any sequence should be allowed. But really, this logic does not need to exist inside this function. It's clearer and more maintainable to keep the loop outside the function.